### DISCUSSION TITLE
Telemetry?

### DISCUSSION THREAD

**Discussion by luigifab** [OpenMage/magento-lts#3956] [General]
Like [popularity-contest](https://popcon.debian.org/) package in Debian, is a telemetry system can be a good idea?

Modules: name, enabled or not, version + OpenMage version, PHP version, DB version and software

I don't suggest to collect store names and urls.

**Comment by Flyingmana** [comment]
primary problem is having an endpoint to send the data to. (someone needs to maintain it)
second problem is filtering out test/staging/local instances.
third problem is people beeing already suspicious even if its only exposing the IP address.

**Comment by Flyingmana** [comment]
packagist already exposes some information

https://packagist.org/packages/openmage/magento-lts/stats

**Comment by pquerner** [comment]
No, I don't like that.
If you want to do it, do it by opt-in with a new module which must be installed consciously.
